frontend/src/metabase/css/core/colors.module.css

```diff
@@ -11,9 +11,9 @@
   --mb-color-text-light: #949aab;
   --color-text-white: #fff;
   --mb-color-bg-black: #2e353b;
-  --color-bg-dark: #93a1ab;
+  --mb-color-bg-dark: #93a1ab;
```

Comment:
For these new colors to work you need to put them in `<GlobalStyles />`. The reason is that we configured CSS module to remove these CSS custom properties, so their values are only accessible at run time, that means all variables here are only applicable for other CSS modules files, but they're accessible at run time i.e. styled components.

We could change the CSS configuration to always emit these CSS custom properties, or put these 2 colors in `GlobalStyles />` as mentioned.

You can see that The appearance color doesn't look the same anymore. The table header background has changed. This is the example where we couldn't access these colors in styled components.
#### This PR
![image](https://github.com/metabase/metabase/assets/1937582/76cc2046-92a8-40b4-8c52-9d8b0e6c0330)

#### Master
![image](https://github.com/metabase/metabase/assets/1937582/d98d9e2c-5b2d-4a6f-9c86-57420cf4083a)

Reply:
I see we're conditionally set the Post CSS custom property setting based on whether or not it's in the SDK context
https://github.com/metabase/metabase/blob/ea7bd55939d0dfd44b3e3e0605721e336c9429e4/postcss.config.js#L20-L24

What about just making it always `true`?